tests/modules/last/dotplot/test.yml

```diff
@@ -1,8 +1,10 @@
 - name: last dotplot test_last_dotplot
-  command: nextflow run ./tests/modules/last/dotplot -entry test_last_dotplot -c ./tests/config/nextflow.config -c ./tests/modules/last/dotplot/nextflow.config
+  command: nextflow run tests/modules/last/dotplot -entry test_last_dotplot -c tests/config/nextflow.config
   tags:
-    - last/dotplot
     - last
+    - last/dotplot
   files:
     - path: output/last/test.png
       md5sum: 6189aaf96f522cdb664869724997bbcd
+    - path: output/last/versions.yml
+      md5sum: 5f270c5a574d077ef3655c275ec600cf
```

Comment:
```suggestion
```

You can delete this for all modules (even if currently auto-generated), makes it easier to update module version in the future (no test failures just because you bump versions)